plugins/modules/route53_crrs.py

```diff
@@ -0,0 +1,504 @@
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+from ansible_collections.amazon.aws.plugins.module_utils.aws.core import (
+    AnsibleAWSModule,
+)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: route53_crrs
+short_description: create, update or delete route53 resource record sets.
+description: create, update or delete route53 resource record sets.
+requirements: [boto3, botocore, python>=3.5]
+options:
+  action:
+    description: >
+      - The action to perform:
+        CREATE  Creates a resource record set that has the specified values.
+        DELETE  Deletes a existing resource record set.
+        UPSERT  If a resource record set doesn't already exist, Route 53
+                creates it. If a resource record set does exist, Route 53
+                updates it with the values in the request.
+    required: true
+    choices: ['CREATE', 'DELETE', 'UPSERT']
+    type: str
+  name:
+    description: >
+      - The name of the record that you want to create, update, or delete.
+    required: true
+    type: str
+  type:
+    description: >
+      - The DNS record type.
+    required: true
+    choices: ['SOA', 'A', 'TXT', 'NS', 'CNAME', 'MX',
+              'NAPTR', 'PTR', 'SRV', 'SPF', 'AAAA', 'CAA']
+    type: str
+  ttl:
+    description: >
+      - The resource record cache time to live (TTL), in seconds.
+    required: true
+    type: int
+  resource_records:
+    description: >
+      - list of the current or new DNS record value(ip address).
+    required: true
+    type: list
+  hosted_zone_id:
+    description: >
+      - The ID of the hosted zone that contains the resource
+        record sets that you want to change.
+    required: true
+    type: str
+  set_identifier:
+    description: >
+      - A unique string to identify each resource record set.
+        it is required if acting upon any of the 'weight', 'region',
+        'failover', 'geo_location', 'multi_value_answer'.
+    type: str
+  weight:
+    description: >
+      - A value that determines the proportion of DNS queries
+        that Amazon Route 53 responds to using the current
+        resource record set.
+    required: true
+    type: int
+  region:
+    description: >
+      - Route 53 selects the latency resource record set that has the
+        lowest latency between the end user and the associated Amazon
+        EC2 Region.
+    required: true
+    choices: [
+      'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2',
+      'ca-central-1', 'eu-west-1', 'eu-west-2', 'eu-west-3',
+      'eu-central-1', 'ap-southeast-1', 'ap-southeast-2',
+      'ap-northeast-1', 'ap-northeast-2', 'ap-northeast-3',
+      'eu-north-1', 'sa-east-1', 'cn-north-1', 'cn-northwest-1',
+      'ap-east-1', 'me-south-1', 'ap-south-1', 'af-south-1',
+      'eu-south-1'
+    ]
+    type: str
+  geo_location:
+    description: >
```

Comment:
This "description" is won't do what you want. Please use a proper `description` and `suboptions`. Grep for `suboptions` in `plugins/modules/` to see how it works.